docs/core/install/includes/linux-not-supported-ubuntu.md

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
I think it would be amazing if DocFx, supported parameter based markdown templating. So that include files could optionally take parameters, from their parent doc. This could really simplify things and remove redundancy - "steps off soapbox".

Reply:
Yeah I know. I think it's been requested for years.